charms/katib-controller/tox.ini

```diff
@@ -66,7 +66,7 @@ description = Check code against coding style standards
 
 [testenv:integration]
 commands = 
-    pytest -v --tb native --asyncio-mode=auto {[vars]tst_path}integration/test_charm.py --log-cli-level=INFO -s {posargs}
+    pytest -vv --tb native --asyncio-mode=auto {[vars]tst_path}integration/test_charm.py --log-cli-level=INFO -s {posargs}
```

Comment:
If we do not need that exrat `v`, I would revert this change to reduce number of changes we do.